godot_project/editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker_controls/atomic_structure_model_validator.gd

```diff
@@ -563,11 +563,11 @@ class SpatialHashGridOverlaps extends SpatialHashGrid:
 							overlapping_atoms[other_atom.structure_context] = []
 						overlapping_atoms[other_atom.structure_context].push_back(other_atom)
 						visited[j] = true
-				if overlapping_atoms.size() <= 1:
-					continue
```

Comment:
This condition is there to prevent false positives, removing it causes a lot of overlap alerts about atoms who are not overlapping with anything 
![image](https://github.com/user-attachments/assets/d87e8dc8-f4ca-47f4-80cf-48d33d1ea337)